### PR TITLE
win32: Add vaDisplayIsValid to .def export list

### PR DESCRIPTION
**Pull request by sivileri**
vaDisplayIsValid is used by apps calling into VAAPI, and we were missing it from the exports list.

Example of usage on [this app source](https://github.com/FFmpeg/FFmpeg/blob/7158f1e64d9b76afea78537a35c465447df0cff8/libavutil/hwcontext_opencl.c#L778): 

Signed-off-by: Sil Vilerino <sivileri@microsoft.com>

### CODE REVIEW

**Comment by sivileri** [comment]
FYI @dvrogozh @XinfengZhang

File: va/libva.def
Comment (dvrogozh) [inline va/libva.def:86]: LGTM considering this function is part of VAAPI public API:
https://github.com/intel/libva/blob/b75818c092367723c76e716c56d3ef9765ce9e51/va/va.h#L445

**Review by dvrogozh — APPROVED** [review]

File: va/libva.def
Comment (sivileri) [reply va/libva.def:86]: @dvrogozh thanks for taking a look! Could a maintainer please help merging it? I don't seem to have enough permissions to do it myself.

image

Comment (dvrogozh) [reply va/libva.def:86]: I am just giving a chance for others to review. I will merge it in tomorrow if no concerns from others unless @XinfengZhang won't do that before me.

**Review by XinfengZhang — APPROVED** [review]

File: va/libva.def
Comment (sivileri) [reply va/libva.def:86]: Sounds good thanks :)

**Pull request merged**
